src/components/organisms/Notifications.tsx

```diff
@@ -79,37 +79,42 @@ export default function Notifications() {
 
   const handleClickMessage = (data: NotificationsData) => {
     setIsOpen(true);
-    setModalData(<><div className={styles.modal__item}>
-      <div className={styles.notifi__itemInfo}>
-        <div
-        className={`${styles.notifi__itemType} ${
-          data.content_type === typeAlarm
-            ? styles["notifi__itemType--alarm"]
-            : data.content_type === typeSnooze
-            ? styles["notifi__itemType--snooze"]
-            : styles["notifi__itemType--out"]
-        }`}
-      >
-        {data.content_type}
-      </div>
-      <p className={styles.notifi__itemDate}>{data.date}</p>
-      </div>
-      <p className={styles.modal__itemContent}>{data.content}</p>
-    </div>
-    <div className={styles.modal__btnArea}>
-      <button className={styles.modal__closebtn} onClick={closeModal}>close</button>
-    </div>
-    </>);
+    setModalData(
+      <>
+        <div className={styles.modal__item}>
+          <div className={styles.notifi__itemInfo}>
+            <div
+              className={`${styles.notifi__itemType} ${
+                data.content_type === typeAlarm
+                  ? styles["notifi__itemType--alarm"]
+                  : data.content_type === typeSnooze
+                  ? styles["notifi__itemType--snooze"]
+                  : styles["notifi__itemType--out"]
+              }`}
+            >
+              {data.content_type}
+            </div>
+            <p className={styles.notifi__itemDate}>{data.date}</p>
+          </div>
+          <p className={styles.modal__itemContent}>{data.content}</p>
+        </div>
+        <div className={styles.modal__btnArea}>
+          <button className={styles.modal__closebtn} onClick={closeModal}>
+            close
+          </button>
+        </div>
+      </>
+    );
 
     // update notifications table
-    // if (!data.is_read) {
-    //   api.post("/notifications", data.id).then((res) => {
-    //     api.get('/notifications').then((res) => {
-    //       setAllNotifiData(res.data);
-    //       setNotifiData(res.data);
-    //     });
-    //   });
-    // }
+    if (!data.is_read) {
+      api.put(`/notifications/${data.id}`).then((res) => {
+        api.get("/notifications").then((res) => {
+          setAllNotifiData(res.data);
+          setNotifiData(res.data);
+        });
```

Comment:
Is this process necessary?
It seems wasteful to retrieve all the notification data again every time user read it.
I think we just need to update the `is_read` by `data.is_read = true`.

Reply:
I need to change font style 'bold' to 'normal' after reading the notification. So I have to get all notifications or re-rendering.